database/exam/V1__exam_create_schema.sql

```diff
@@ -0,0 +1,582 @@
+/***********************************************************************************************************************
+  File: V1__exam_create_schema.sql
+
+  Desc: create the exam database schema
+
+***********************************************************************************************************************/
+-- Server version	5.6.27-log
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+CREATE SCHEMA IF NOT EXISTS exam
+  CHARACTER SET utf8
+  COLLATE utf8_unicode_ci;
+
+USE exam;
```

Comment:
Historically, we've used the TDS_Build project to create the database schemas for the TDS applications.  But we could certainly move the exam schema creation to the `TDS_ExamService` project and update its readme for creating the schema.